schemas/triniti/news/command/inspect-article-seo/1-0-0.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<pbj-schema xmlns="http://gdbots.io/pbj/xsd"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://gdbots.io/pbj/xsd http://gdbots.io/pbj/xsd/schema.xsd">
+
+  <schema id="pbj:triniti:news:command:inspect-article-seo:1-0-0">
+    <fields>
+      <field name="node_ref" type="node-ref" required="true"/>
+      <field name="providers" type="string" format="slug" rule="set">
+        <description>
+          The name of the search engines to get the stats from, e.g. google, bing, badoo, etc. When not provided you should call all providers.
```

Comment:
@digitalartcrew lets do:

The search engines to inspect, e.g. google, bing, baidu, etc. When not provided you should inspect all providers.